### PR TITLE
Bump to LBT v5.7.0

### PR DESCRIPTION
**Pull request by staticfloat**
This release of libblastrampoline contains two new features: ILP64 Accelerate support, and LAPACK_jll support.

### CODE REVIEW

**Review by inkydragon — COMMENTED** [review]
### Check list

doc: https://github.com/JuliaLang/julia/blob/master/doc/src/devdocs/build/build.md#update-the-version-number-of-a-dependency

Version numbers:
- [ ] `deps/$(libname).version`: `LIBNAME_VER`, `LIBNAME_BRANCH`, `LIBNAME_SHA1` and `LIBNAME_JLL_VER`
- [x] `stdlib/$(LIBNAME_JLL_NAME)_jll/Project.toml`: `version`

Checksum:
- [x] `deps/checksums/$(libname)`

Patches:
- no patch

File: stdlib/libblastrampoline_jll/Project.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-version = "5.4.0+0"
+version = "5.7.0+0"
 
 [deps]
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-julia = "1.8"
+julia = "1.10"
```
Comment (ViralBShah) [inline stdlib/libblastrampoline_jll/Project.toml:10]: We should make this 1.9, so that it can be backported and made available in 1.9.1.

Comment (staticfloat) [reply stdlib/libblastrampoline_jll/Project.toml:10]: When we backport, we can set it accordingly.  I don't want to set it to `1.9`, and then it's only available in `1.9.1`

**Comment by DilumAluthge** [comment]
```
ERROR: The working directory is dirty.
--
Output of git status:
HEAD detached at 46adde9d8d7
Untracked files:
(use "git add <file>..." to include in what will be committed)
 
deps/checksums/blastrampoline-2272604bfb10b9e8a3ae5f1a4569899b99251a65.tar.gz/
 
nothing added to commit but untracked files present (use "git add" to track)
🚨 Error: The command exited with status 1
```

**Comment by DilumAluthge** [comment]
@staticfloat Cam you update the source checksum for LBT?

**Comment by ViralBShah** [comment]
> @staticfloat Cam you update the source checksum for LBT?

Fixed in https://github.com/JuliaLang/julia/pull/49396

**Pull request merged**